### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by ChristophConrads-deepstreamHub**

### CODE REVIEW

File: src/tutorials/integrations/other-openauth/readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ---
 title: Open Auth
 draft: true
----
\ No newline at end of file
+---
```
Comment (yasserf) [inline src/tutorials/integrations/other-openauth/readme.md]: What changed here? Line endings?

File: src/tutorials/integrations/frontend-react/readme.md [hunk 2]
```diff
@@ -38,12 +38,12 @@ Next: connect to the server, log in and register the client instance with deepst
 
 ```javascript
 const deepstream = require('deepstream.io-client-js')
-const DeepstreamMixin = require('deepstream.io-tools-react')
+const deepstreamMixin = require('deepstream.io-tools-react')
```
Comment (yasserf) [inline src/tutorials/integrations/frontend-react/readme.md]: This can't be lower cased since its a class

Comment (yasserf) [inline src/tutorials/integrations/frontend-react/readme.md]: And all other usages in this code sample

File: src/tutorials/core/security-overview/readme.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ Often it's easier and more performant though to leave SSL termination to a load
 
 ## Authentication
 Every incoming connection needs to pass an authentication step. This happens when the client calls `login( data, callback )`.
-Deepstream comes with three built-in authentication mechanisms:
+deepstream comes with three built-in authentication mechanisms:
 
-- [none](#) allows every connection. Choose this option for public sites that don't require access controls.
-- [file](#) reads authentication data from a static file. This is a good choice for public read / private write use cases, e.g. sports result pages that let every user visit, but only a few backend processes update the result.
-- [http](#) contacts a configurable HTTP webhook to ask if a user is allowed to connect. This is the most flexible option as it allows you to write a tiny http server in any language that can connect to databases, active directories, oAuth providers or whatever else your heart desires.
+- [none](https://deepstream.io/tutorials/core/auth-none/) allows every connection. Choose this option for public sites that don't require access controls.
```
Comment (yasserf) [inline src/tutorials/core/security-overview/readme.md]: 👍

File: src/tutorials/integrations/frontend-react/readme.md (hunk 2 above)
Comment (ChristophConrads-deepstreamHub) [inline src/tutorials/integrations/frontend-react/readme.md]: Shall I capitalize `const deepstream = ...` as well? The pull request does not change the line but the formatting is inconsistent.

File: src/tutorials/integrations/other-openauth/readme.md (hunk 1 above)
Comment (ChristophConrads-deepstreamHub) [inline src/tutorials/integrations/other-openauth/readme.md]: Yes.

File: src/tutorials/integrations/frontend-react/readme.md (hunk 2 above)
Comment (yasserf) [inline src/tutorials/integrations/frontend-react/readme.md]: nope. It's a bit wierd, but the DeepstreamReactMixin is a class and setDeepstream is a static method. deepstream however is a function and hence is lower case

**Pull request merged**